app/src/main/res/values-es/strings.xml

```diff
@@ -0,0 +1,149 @@
+     <string name="ram">RAM</string> 
+     <string name="reset">Resetear</string> 
+     <string name="reset_message"> 
+         Esta operación borrará todos los datos almacenados de la aplicación eneste dispositivo, y no puede deshacerse.
+     </string> 
+     <string name="restart">Reiniciar</string> 
+     <string name="right">Derecha</string> 
+     <string name="root">Raíz</string> 
+     <string name="rss_feed">Feed RSS</string> 
+     <string name="sdk" translatable="false">SDK</string> 
+     <string name="search_with_keyboard">Buscar con teclado</string> 
+     <string name="shortcut_count">Número de accesos directos (Url/Contacto)</string> 
+     <string name="show_city">Mostrar nombre de la ciudad en los datos meteorológicos</string> 
+     <string name="something_went_wrong">Algo ha ido mal</string> 
+     <string name="source_code">Código fuente</string> 
+     <string name="star">Estrella</string> 
+     <string name="support">Soporte</string> 
+     <string name="support_message">Apoya al desarrollador para mantener esta aplicacion viva y gratuita de por vida.\n 
+         \n<li><b>Donar:</b> Si es posible, haz donaciones directas para ayudar y motivar al desarrollador.</li> 
+         \n<li><b>Amazon:</b> Usa el link de afiliado al comprar en Amazon para que el desarrollador sea recompensado</li> 
+         \n<li><b>Estrella:</b> Marca el repositorio con una estrella para mostrar tu apoyo a la aplicación./li> 
+     </string> 
+     <string name="system_stats">Información del sistema</string> 
+     <string name="telegram_group">Grupo de Telegram</string> 
+     <string name="temperature">Temperatura</string> 
+     <string name="temp_unit">Unidad de temperatura</string> 
+     <string name="time_date">Hora - Fecha</string> 
+     <string name="time_format">Formato de hora</string> 
+     <string name="todo">Tareas</string> 
+     <string name="todo_count">Número de tareas en la pantalla de inicio</string> 
+     <string name="todo_manager">Gestor de tareas</string> 
+     <string name="todo_manager_lock">Bloquear gestor de tareas</string> 
+     <string name="total">Total</string> 
+     <string name="twelve">12 Horas</string> 
+     <string name="twenty_four">24 Horas</string> 
+     <string name="uid" translatable="false">UID</string> 
+     <string name="unable_to_launch">Fallo al abrir</string> 
+     <string name="uninstall">Desinstalar</string> 
+     <string name="update">Actualizar</string> 
+     <string name="url">Url</string> 
+     <string name="used">Usado</string> 
+     <string name="version">Versión</string> 
+     <string name="wallpaper_change_success">Fondo de pantalla cambiado con éxito</string> 
+     <string name="weather">Tiempo</string> 
+     <string name="welcome">Bienvenide</string> 
+     <string name="welcome_description">Bienvenide a Lunar Launcher. Esta es la primera vez que abres la aplicación; si es la primera vez que la usas, <b>lee estos textos antes de proceder<\b>.
```

Comment:
Here, at the end of this line, use forward slash instead of a backward slash for bold's end tag. (~~<\b>~~ --> `</b>`)